app/policies/log_policy.rb

```diff
@@ -0,0 +1,24 @@
+class LogPolicy
+  attr_reader :user, :log
+
+  def initialize(user, log)
+    @user = user
+    @log = log
+  end
+
+  def destroy?
+    return false unless log && user
+
+    # Can only delete editable logs
+    return false unless log.collection_period_open?
+
+    # Only delete logs with answered questions
+    return false unless log.in_progress? || log.completed?
+
+    # Data coordinators and support users can delete any log
+    return true if user.data_coordinator? || user.support?
```

Comment:
This line as it is would allow any data coordinator of any organisation to delete a log. I suggest we scope this to the relevant organisation, but also update this finder in the controller so that it can only find relevant logs:

```ruby
    @log = LettingsLog.visible.find(params[:id])
```

Reply:
@bibblobcode Sorry, I was suggesting that those lines in the controller action might want to change, so that you can be defensive against enumerating IDs or permission failures.